projpicker/web/index.html

```diff
@@ -0,0 +1,337 @@
+            institute-environmental-spatial-analysis/index.php">Institute of
+            Environmental and Spatial Analysis</a> (IESA) at the University of
+            North Georgia.
+            </p>
+        </div>
+    </div>
+</body>
+<script>
+    var map = L.map(
+        'map', {
+            center: [34.2347566, -83.8676613],
+            crs: L.CRS.EPSG3857,
+            zoom: 5,
+            zoomControl: true,
+            preferCanvas: false,
+        }
+    );
+
+    var tileLayer = L.tileLayer(
+        'https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png', {
+            'attribution': 'Data by \u0026copy; \u003ca href="http://openstreetmap.org"\u003eOpenStreetMap\u003c/a\u003e, under \u003ca href="http://www.openstreetmap.org/copyright"\u003eODbL\u003c/a\u003e.',
+            'detectRetina': false,
+            'maxNativeZoom': 18,
+            'maxZoom': 18,
+            'minZoom': 0,
+            'noWrap': true,
+            'opacity': 1,
+            'subdomains': 'abc',
+            'tms': false
+        }
+    ).addTo(map);
+
+    // FeatureGroup is to store editable layers.
+    var geomsLayer = new L.geoJSON().addTo(map);
+
+    new L.Control.Draw({
+        position: 'topleft',
+        draw: {
+            'polyline': {
+                'allowIntersection': false
+            },
+            circle: false,
+            circlemarker: false,
+            polyline: false
+        },
+        edit: {
+            'poly': {
+                'allowIntersection': false
+            },
+            'featureGroup': geomsLayer
+        },
+    }).addTo(map);
+
+    map.on(L.Draw.Event.CREATED, async function (e) {
+        geomsLayer.addLayer(e.layer);
+        await sendQuery();
+
+        await getData('/projdata')
+            .then(data => populateCRSList(Object.keys(data)))
+    });
+
+    function populateCRSList(crsKeys) {
+        const list = document.getElementById('crslist');
+        while (list.firstChild) {
+            list.removeChild(list.firstChild);
+        }
+        const listItems = crsKeys.map( function(element) {
+                return `<li tabindex='1' id='${element}' onclick='onCRSSelect()'>${element}</li>` });
+
+        list.innerHTML = listItems.join('');
+    }
+
+    async function getData(url) {
+      var response = await fetch(url);
+      return response.json();
+    }
+
+    async function onCRSSelect() {
+        var selectedId = document.activeElement.id;
+        console.log(selectedId);
+        var data = await getData('/projdata');
+        console.log(data)
+        selectedCRS = data[selectedId];
+
+        var s = selectedCRS.south_lat
+        var n = selectedCRS.north_lat
+        var w = selectedCRS.west_lon
+        var e = selectedCRS.east_lon
+
+        var coors = [[[w, n], [e, n], [e, s], [w, s]]]
+
+        var crsInfo = document.getElementById('crs-info')
+        var crsInfoList = [];
+            crsInfoList.push(`<tr><td>Name</td><td>Value</td></tr>`)
```

Comment:
Using literal tag strings is generally not a good idea. Instead, we use `document.createElement()`